README.md

```diff
@@ -6,7 +6,6 @@
 
 [![PyCall](http://pkg.julialang.org/badges/PyCall_0.5.svg)](http://pkg.julialang.org/?pkg=PyCall&ver=0.5)
```

Comment:
Did you mean to delete this line and keep the 0.7 line?

Reply:
Actually, I meant to drop the line showing 0.7 failing, as it gives the impression 0.7 and 1.0 is failing... while I believe it to be false. I'm not sure if how can be modified to show 0.7 (and/or at least 1.0 succeeding).

I don't care too much about showing 0.5 not failing, it's your call to drop that one or not. My understanding is that if you up the requirements, it shows the last Julia version working for 0.5, and I can go either way, keep it or not.